src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -36,18 +34,17 @@ class DoctrineChoiceLoader extends AbstractChoiceLoader
      *
      * @param string $class The class name of the loaded objects
      */
-    public function __construct(ObjectManager $manager, string $class, IdReader $idReader = null, EntityLoaderInterface $objectLoader = null)
-    {
-        $classMetadata = $manager->getClassMetadata($class);
-
+    public function __construct(
+        private readonly ObjectManager $manager,
+        string $class,
```

Comment:
we can still declare the property here if we want

Reply:
Yes, but not as `readonly` then because we change its value within the constructor.